src/compiler/commandLineParser.ts

```diff
@@ -141,6 +141,7 @@ namespace ts {
                     "es2017.sharedmemory": "lib.es2017.sharedmemory.d.ts",
                     "es2017.string": "lib.es2017.string.d.ts",
                     "es2017.intl": "lib.es2017.intl.d.ts",
+                    "es2017.typedarrays": "lib.es2017.typedarrays.d.ts",

```

Comment:
mind updating it in https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Compiler%20Options.md as well.

Reply:
https://github.com/Microsoft/TypeScript-Handbook/pull/680